rust/cjdns_sys/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 build = "build.rs"
 
 [dependencies]
-sodiumoxide = { git = "https://github.com/cjdelisle/sodiumoxide", rev = "9f6a18d40a4db253edfebac9f2ce5c22d09b1f47", version = "0.2", default-features = false, features = ["std"] }
+sodiumoxide = { git = "https://github.com/cjdelisle/sodiumoxide", rev = "f79b6656b84f02b049470f9ad556a6a0c9980bd7", version = "0.2", default-features = false, features = ["std"] }
```

Comment:
Could you please provide a link to where I can find `f79b6656b84f02b049470f9ad556a6a0c9980bd7` ?

Reply:
https://github.com/cjdelisle/sodiumoxide/pull/1/commits/f79b6656b84f02b049470f9ad556a6a0c9980bd7

Reply:
It's coming from this (unmerged) PR https://github.com/cjdelisle/sodiumoxide/pull/1